draft-ietf-quic-recovery.md

```diff
@@ -354,7 +355,7 @@ frequently observable.
 ## Estimating smoothed_rtt and rttvar {#smoothed-rtt}
 
 smoothed_rtt is an exponentially-weighted moving average of an endpoint's RTT
-samples, and rttvar is the variation in the RTT samples, estimated using a
+samples, and rttvar represents the variation in the RTT samples, estimated using a
```

Comment:
(if it's "estimated", then it's not "is")

Reply:
It is if the definition of the term is an representation of an estimate, which is what it is.

Reply:
I don't think the comma is appropriate if the claim is that rttvar is defined to be the estimated variation of the RTT sample.

Reply:
I can change this to "and rttvar is the mean variation in the RTT samples." if that's preferable?